src/bin/cli/polkadot/call.rs

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Code stolen from [cargo contract](https://github.com/paritytech/cargo-contract/blob/632959c5e37e7d86bf09e1966e8c6637321cd9aa/crates/cargo-contract/src/cmd/call.rs).

I'm no expert but strictly this might even constitute a GPL violations, as Solang is not GPL licensed.